app/src/main/java/com/framgia/englishconversation/screen/imagedetail/ImageDetailViewModel.java

```diff
@@ -39,19 +40,30 @@ public void onItemTimelineClick(TimelineModel item) {
                     //TODO
                 }
 
+                /**
+                 * @param item truyền vào khi người dùng click vào layout header item
+                 * check điều kiện nếu userModel từ profile gừi sang trùng với user người tạo
+                 * post thì không điều hướng sang activity profile mới
+                 */
                 @Override
                 public void onItemUserNameClick(TimelineModel item) {
+                    if (mTimelineUser != null && mTimelineUser.getId()
+                            .equals(item.getCreatedUser().getId())) {
+                        return;
```

Comment:
ERROR: 'if' child have incorrect indentation level 24, expected level should be one of the following: 16, 20.